### PR TITLE
Better error handling

### PR DESCRIPTION
**Pull request by GmBodhi**
**Please describe the changes this PR makes and why it should be merged:**

I've added the property `ephemeral` to the `CommandInteraction` class. 
This is useful for throwing an error while fetching an ephemeral response.

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

**Comment by vBread** [comment]
The typings need to be updated as well.

File: src/structures/CommandInteraction.js [hunk 1]
```diff
@@ -32,6 +32,12 @@ class CommandInteraction extends Interaction {
      * @type {boolean}
      */
     this.deferred = false;
+    
+    /**
+     * Whether the reply to this interaction is ephemeral
+     * @type {boolean}
+     */
+    this.ephemeral = false;
```
Comment (SpaceEEC) [inline src/structures/CommandInteraction.js:40]: Maybe initialize it with `null` since the interaction wasn't replied to yet?

Comment (GmBodhi) [reply src/structures/CommandInteraction.js:40]: Okay

**Comment by kyranet** [comment]
This PR has conflicting files, please rebase it on top of fixing the requested changes above?

**Comment by iCrawl** [comment]
Superseeded by #5892

**Pull request closed without merging**